parsers/miniseq/sample_parser.py

```diff
@@ -295,11 +295,13 @@ def _parse_samples(sample_sheet_file):
 
         new_sample_dict = deepcopy(sample_dict)
         new_sample_name = new_sample_dict['sampleName']
-        # Some versions of the illumina *seq software has description fields, and others do not
+        # Some versions of the illumina *seq software has Description fields, and others do not
         # If they do, we need to include the description field here (or else we end up with duplication of fields)
         # If they don't we give it a blank description
         new_sample_desc = ''
         if 'Description' in new_sample_dict:
+            new_sample_desc = new_sample_dict['Description']
+        if 'description' in new_sample_dict:
```

Comment:
What's effect does the difference between the capital `D` and small `d` description fields have?

Reply:
Depends how it comes off the sample sheet. I have seen both happen on the sequencers now, and I don't know if it's something weird with illumina software, or if it is operator error, but this covers the bases and makes sure that if something weird happens where two different description fields appear, it can continue.